app/views/index.scala.html

```diff
@@ -1,12 +1,12 @@
 @(isLoggedIn: Boolean, name: String, isJudgmentUser: Boolean)(implicit messages: Messages, request: RequestHeader)
-@defining("The National Archives Transfer Digital Records") { title =>
+@defining("The National Archives") { title =>
```

Comment:
Can remove this line and add the `"The National Archives"` directly" to the `@main`